content/docs/reference/spec/buildpack-api.md

```diff
@@ -52,7 +53,7 @@ if [ -f requirements.txt ]; then
   echo "Python Buildpack"
   exit 0
 else
-  exit 1
+  exit 100
```

Comment:
Do we use exit code 100?

Reply:
[Per spec](https://github.com/buildpacks/spec/blob/main/buildpack.md#detection), 100 is failed detection (as in, this buildpack does not apply to this app) and other non-zero indicates an error during detection.

Reply:
Ah, didn't catch that. Yup, should be okay to switch to 100 then.